tests/summarycode/data/tallies/full_tallies/tallies_key_files-details.expected.json-lines

```diff
@@ -20,11 +20,11 @@
         "warnings": [],
         "extra_data": {
           "system_environment": {
-            "operating_system": "linux",
```

Comment:
Minor nitpick for next time, we can avoid  this churn on force regen by either removing the header if not necessary for test purposes, or just not commit these changes if possible. Don't have to change anything for now.